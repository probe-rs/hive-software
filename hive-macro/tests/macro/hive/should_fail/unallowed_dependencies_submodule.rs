#[hive_macro::hive]
pub mod tests {
    mod a {
        mod b {
            #![allow(unused)]
            use hive_test;
            use std;
            use {
                std::sync,
                std::time,
                {quote, std::alloc, std::io},
            };
        }
    }
}

fn main() {}
