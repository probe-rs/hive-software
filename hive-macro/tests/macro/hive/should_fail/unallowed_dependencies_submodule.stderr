error: The dependency 'quote' is not allowed in Hive test modules

         = help: The Hive testserver does not support this dependency. If it is required nonetheless, please file an issue

  --> tests/macro/hive/should_fail/unallowed_dependencies_submodule.rs:11:18
   |
11 |                 {quote, std::alloc, std::io},
   |                  ^^^^^
